src/client/pydaos/pydaos_core.py

```diff
@@ -124,14 +132,14 @@ def genoid(self, cid):
             raise PyDError("failed to generate object identifier", ret)
         return ObjID(hi, lo)
 
-    def newkv(self, cid = ObjClassID.OC_SX):
+    def newkv(self, cid=ObjClassID.OC_SX):
         oid = self.genoid(cid)
         return KVObj(self.coh, oid, self)
 
     def kv(self, oid):
         return KVObj(self.coh, oid, self)
 
-    def rootkv(self, cid = ObjClassID.OC_SX):
+    def rootkv(self, cid=ObjClassID.OC_SX):
```

Comment:
(pylint-missing-docstring) Missing method docstring
(pylint-no-member) Instance of 'Enum' has no 'OC_SX' member